api/src/main/java/org/openmrs/module/aijar/tasks/GenerateUniqueIdentifierCodeTask.java

```diff
@@ -0,0 +1,235 @@
+package org.openmrs.module.aijar.tasks;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.context.Context;
+import org.openmrs.scheduler.tasks.AbstractTask;
+
+public class GenerateUniqueIdentifierCodeTask extends AbstractTask {
+    private Log log = LogFactory.getLog(this.getClass());
+    @Override
+    public void execute() {
+        log.info("method executing started");
+        generateUICForPatientsWithout();
+    }
+
+    private void generateUICForPatientsWithout() {
+
+        try {
+            String queryString = "insert into patient_identifier (patient_id,identifier_type,date_created,preferred,location_id,creator ,changed_by,voided_by,identifier,uuid) \n"
```

Comment:
Create a constant in the Aijar Constants and put the Query there. Then reference the query

Reply:
Done